.circleci/config.yml

```diff
@@ -1188,6 +1224,7 @@ jobs:
           path: fastlane/test_output
       - store_artifacts:
           path: fastlane/test_output
+      - slack-notify
```

Comment:
we shouldn't run this on `lint`

Reply:
This actually runs on `main`. I think the chances of merging something breaking this would be low, but better to keep it. Lmk if you think otherwise!